src/v/redpanda/admin_server.cc

```diff
@@ -2640,6 +2648,18 @@ void admin_server::register_partition_routes() {
       });
 }
 
+ss::future<ss::json::json_return_type>
+admin_server::trigger_on_demand_reconfiguration_handler(
```

Comment:
nit: should be "on demand rebalancing" - when I hear "reconfiguration" I think about a single partition